help/en/html/apps/PanelPro/Sidebar

```diff
@@ -13,8 +13,8 @@
 <li><a href="../../../package/jmri/jmrit/display/LayoutEditor.shtml">Layout Editor</a>
 <li><a href="../../../package/jmri/jmrit/display/SwitchboardEditor.shtml">Switchboard Editor</a>
 <li><a href="FAQ.shtml">FAQ</a>
-<li><a href="http://jmri.org/community/examples/Gallery.shtml">User's Examples</a>
-<li><a href="http://jmri.org/community/examples/Modules.shtml">Modular Layout Example</a>
+<li><a href="https://jmri.org/community/examples/Gallery.shtml">User's Examples</a>
+<li><a href="https://jmri.org/community/examples/Modules.shtml">Modular Layout Example</a>
```

Comment:
Does these links need to start with the web server? Isn't `href="/community/examples/Gallery.shtml"` enough? If the web site is cloned to another web server, for example a local web server during development of the web pages, it's easier to make it work if the links are without reference to jmri.org.

Reply:
Thanks for the suggestion, but the difficulty here is in that the ``community`` part is part of the ``website`` project while the normal Help html is in the ``JMRI`` project. Cloning the whole thing seems not the most common use case, and I do not know how to replace this with a relative path betwen the two. Unless you set up a real web server it won't work. Adding the ``s`` to ``http`` is just a little maintenance as I came across the link.
The big files displayed on the Gallery are not part of the distibution either.